core/src/services/gcs/core.rs

```diff
@@ -250,6 +250,48 @@ impl GcsCore {
         self.send(req).await
     }
 
+    pub async fn gcs_delete_objects(
+        &self,
+        paths: Vec<String>,
+    ) -> Result<Response<IncomingAsyncBody>> {
+        let uri = "/batch/storage/v1/".to_string();
```

Comment:
Seems we are missing the endpoint for batch.

Reply:
Is endpoint required? I did not see it in the example:
![image](https://user-images.githubusercontent.com/58715567/234888118-e4bddda2-cf3d-4071-80fe-ada7687ea441.png)

Reply:
All API must be sent to `self.endpoint`.

Reply:
I see.